app/controllers/distributions_controller.rb

```diff
@@ -1,4 +1,4 @@
-class DistributionsController < ApplicationController
+class DistributionsController < OBSController
```

Comment:
this is similar to the `SearchController`, why is this in a different commit and the other one not? :thinking:

Reply:
The initial idea was to remove the dependency of the obs connection for the distribution controller. As far as I understand it, we want to make /explore the default route and will remove the search box from /distributions. 

5fd084f is just a temporary fix until the route is changed and the search box removed. I can also move this to the other commit. I only realized the dependency is still required because of the tests and added this commit to pass them.

Reply:
@alexandergraul 

> I only realized the dependency is still required because of the tests and added this commit to pass them.

If that is the reason for it to be in a different commit, then for sure you should move it to the first one :rofl:

Reply:
It is the reason I realized the dependency still exists, but it is still a temporary thing until we remove the search bar completely from /distributions (which is what #265 is about).

I am not against moving it to the other commit, just want to lay out my motivation.